delfin/common/constants.py

```diff
@@ -62,8 +62,9 @@ class ControllerStatus(object):
 class StorageType(object):
     BLOCK = 'block'
     FILE = 'file'
+    MIXED = 'mixed'
```

Comment:
Hope there are some exisiting storage type like UNITY and OCEANSTOR NEEd an update . If changes are driver specific we can probably create an issue!

Reply:
Created an issue #485 to track this and update drivers

Reply:
block_file, or mixed which is better?